src/main/index.ts

```diff
@@ -5,29 +5,12 @@
 
 // Main process
 
-import * as Mobx from "mobx";
-import { spawn } from "node-pty";
-import * as LensExtensionsCommonApi from "../extensions/common-api";
-import * as LensExtensionsMainApi from "../extensions/main-api";
 import { getDi } from "./getDi";
 import startMainApplicationInjectable from "./start-main-application/start-main-application.injectable";
+import { Mobx, LensExtensions, Pty } from "./extension-api";
 
 const di = getDi();
 
 void di.inject(startMainApplicationInjectable);
```

Comment:
Looks like an inject-time side-effect, because function call is missing. Also, shouldn't we invoke the new `startApp` here in `index.ts` to guarantee identical behaviour with clients of "library"?

Reply:
I believe this is already fixed.